k8s/namespaces/fees-pay/ccpay-dlq-payment-job/prod-00.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: helm.fluxcd.io/v1
```

Comment:
you need to add this file to k8s/prod/cluster-00-overlay/kustomization.yaml

in the `patchesStrategicMerge`

Reply:
done